File: module-1/Week3_Review/exercise-final/dotnet/PetElevator.Tests/CustomerTests.cs
using Microsoft.VisualStudio.TestTools.UnitTesting;
using PetElevator.CRM;
using System;
using System.Collections.Generic;

namespace PetElevator.Tests
{
    [TestClass]
    public class CustomerTests
    {
        [TestMethod]
        public void GetBalanceDue_Test()
        {
            Customer customer = new Customer("Test", "Testerson");
            Dictionary<string, double> services = new Dictionary<string, double>()
            {
                {"Grooming", 45 },
                {"Sitting", 120 },
                {"Walking", 37.50 }
            };

            double due = customer.GetBalanceDue(services);

            Assert.AreEqual(202.50, due);
        }
    }
}

File: module-1/Week3_Review/exercise-final/java/src/test/java/com/techelevator/crm/CustomerTests.java
package com.techelevator.crm;

import org.junit.Test;

import java.util.HashMap;
import java.util.Map;

import static org.junit.Assert.assertEquals;

public class CustomerTests {

    @Test
    public void getBalanceDue() {
        Customer customer = new Customer("Test", "Testerson");

        Map<String,Double> services = new HashMap<>();
        services.put("Grooming", 45.00);
        services.put("Sitting", 120.00);
        services.put("Walking", 37.50);
        double due = customer.getBalanceDue(services);

        assertEquals("The balance for services rendered is not correct.",202.50, due,0.0);
    }
}
